### PR TITLE
chapter3

### PR DESCRIPTION
**Pull request by mirulzaei**
ready

### CODE REVIEW

**Comment by mirulzaei** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "2.7.1"
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+gem "sqlite3", "~> 1.4"
```
Comment (ngocvt-0484) [inline Gemfile]: please use gem mysql2

Comment (mirulzaei) [reply Gemfile]: done

**Comment by mirulzaei** [comment]
ready

**Comment by vanvtt-0952** [comment]
commented
plz, rebase 2 commit

**Comment by mirulzaei** [comment]
ready

**Comment by ngocvt-0484** [comment]
please change 2 commits to 1 commit
you can use
git reset --soft origin/master
git add .
git commit -m "Chapter 3"
git push origin -f

**Comment by ngocvt-0484** [comment]
commented

**Comment by mirulzaei** [comment]
ready

**Comment by vanvtt-0952** [comment]
merged

**Pull request merged**